### ISSUE TITLE
The final output dim of Discriminator seems incorrect.

### ISSUE THREAD

**Issue by BeautifulBeer** [AlanDongMu/TimeGAN_PytorchRebuild#6]
I think the init function of the discriminator in "networks.py" might be changed as below,

    def __init__(self, para):
        super(Discriminator, self).__init__()
        rnn_cell = get_rnn_cell(para['module'])
        self.rnn = rnn_cell(input_size=para['hidden_dim'], hidden_size=para['hidden_dim'], num_layers=para['num_layer'],
                            batch_first=True)
        self.fc = nn.Linear(para['hidden_dim'], 1) #  para['hidden_dim'] -> 1
        self.sigmoid = nn.Sigmoid()

Thank you
